airflow/providers/google/marketing_platform/hooks/display_video.py

```diff
@@ -170,3 +184,69 @@ def download_line_items(self, request_body: Dict[str, Any]) -> List[Any]:
             .execute(num_retries=self.num_retries)
         )
         return response["lineItems"]
+
+    def create_sdf_download_operation(self, body_request: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Creates an SDF Download Task and Returns an Operation.
+        :param body_request: Body request.
+        :type body_request: Dict[str, Any]
+
+        More information about body request n be found here:
+        https://developers.google.com/display-video/api/reference/rest/v1/sdfdownloadtasks/create
+        """
+
+        result = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .sdfdownloadtasks()
+            .create(body=body_request)
+            .execute(num_retries=self.num_retries)
+        )
+        return result["name"]
+
+    def get_sdf_download_operation(self, name: Dict[str, Any]):
+        """
+        Gets the latest state of an asynchronous SDF download task operation.
+        :param name: The name of the operation resource.
+        :type name: str
+        """
+
+        result = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .sdfdownloadtasks()
+            .operation()
+            .get(name=name)
+            .execute(num_retries=self.num_retries)
+        )
+        return result
+
+    def download_media(self, resource_name: Dict[str, Any]):
+        """
+        Downloads media.
+        :param resource_name: of the media that is being downloaded.
+        :type resource_name: str
+        """
+
+        request = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .media()
+            .download_media(resource_name=resource_name)
+        )
+        return request
+
+    def get_operation(self, operation: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        :param operation: result of method get_sdf_download_operation,
+        :type operation: Dict[str, Any]
+        """
+
+        operation_name = operation.get("name")
+
+        operation = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .projects()
+            .locations()
+            .operations()
+            .get(name=operation_name)
+            .execute()
+        )
+        return operation["resourceName"]
```

Comment:
I think we should return the operation object

Reply:
I suggested this documentation https://developers.google.com/display-video/api/guides/downloading-sdfs/download.